student/forms.py

```diff
@@ -9,6 +9,16 @@ class Meta:
         model = Student
         fields = ["full_name", "email", "date_of_birth", "phone_number", "educational_institution",
                   "major", "about", "graduation_date"]
+        widgets = {
```

Comment:
There are a few issues with this implementation:

1. In general this kind of styling info does not belong in the code like this - its better be done in the template files. Its was a but inconvenient to do that in Django up to version 4.0 so I understand why you didn't.
2. There is also a massive amount of code repetition here you might want to avoid in the future.
3. By manually writing all the widgets you increase your maintenance costs because youll have ti make sure you keep it in sync in the future
4. By implementing things like this just for one particular form, you'll have to keep re-implementing for each and every form.

It might not be simple for you to fix all thosae issues for now, so I will merge the code as-is